Pages/Cart/cart.html

```diff
@@ -0,0 +1,126 @@
+            ><i class="fa fa-shopping-cart icon-cart"></i
+          ></a>
+          <span class="icon-text">Cart</span>
+        </div>
+      </div>
+    </nav>
+
+    <div class="container-flex flex-center">
+      <h2 class="font-xl">MyCart(3)</h2>
+    </div>
+    <br />
+
+    <main class="container-flex">
+      <section class="cart-item">
+        <div class="container-flex flex-wr">
+          <div class="horizontal-card-container container-flex">
+            <div class="image-container badge-container">
+              <img
+                src="https://m.media-amazon.com/images/I/51FdLmNuPSL._SL1000_.jpg"
+                class="img"
+                alt="guitar image"
+              />
+            </div>
+            <div class="container-flex flex-column flex-center">
+              <div class="card-title container-flex">
+                <p class="font-medium margin-m1 padding-p1">
+                  Gibson Acoustic Guitar
+                </p>
+              </div>
+              <div class="container-flex flex-column">
+                <p class="card-description margin-m1 padding-p2">
+                  Wooden Guitar with 4 pick ,extra strings,bag and strap
+                </p>
+                <p class="card-price margin-m1 padding-p2">
+                  Rs. 1399 <strike class="card-strikeprice">Rs. 2999</strike>
+                  <span class="card-discount">(30% Off)</span>
+                </p>
+                <div>
+                  <div>
+                    <span class="quantity quantity-decrease">-</span>
+                    <span class="quantity">1</span>
+                    <span class="quantity quantity-increase">+</span>
+                  </div>
+                  <button class="btn-shopping-cart btn-small font-medium">
+                    Add to Wishlist
+                  </button>
+                  <button
+                    class="btn-shopping-cart btn-small font-medium btn-secondary"
+                  >
+                    Remove
+                  </button>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="cart-price">
+        <div class="container-flex price-container flex-column">
+          <h1 class="font-xl">Price details</h1>
+
+          <table>
+            <tr>
+              <td class="font-medium">Price (2 items)</td>
+              <td class="font-medium">4999</td>
+            </tr>
+            <tr>
+              <td class="font-medium">Discount</td>
+              <td class="font-medium">1999</td>
+            </tr>
+            <tr>
+              <td class="font-medium">Delivery Charges</td>
+              <td class="font-medium">100</td>
+            </tr>
+          </table>
+          <hr class="someClass" />
+
+          <table>
+            <tr>
+              <td class="font-medium">Total Price</td>
+              <td class="font-medium">6999</td>

```

Comment:
Add the total price in bold and add some margin and padding-left. otherwise looks good to merge for me

Reply:
sure